src/app/features/fss-search/fss-search-results/fss-search-results.component.html

```diff
@@ -41,11 +42,23 @@
         </ng-container>
         <ng-container ukhoColumnDef="Download">
           <th ukho-header-cell *ukhoHeaderCellDef>Download</th>
-          <td ukho-cell *ukhoCellDef="let element"><div class="fileDownload" [attr.rel]="element.FileLink" tabindex="0" role="button" attr.aria-label="Download File for {{element.FileName}}"><i class="fa fa-download fa-1x"></i></div></td>
+          <td ukho-cell *ukhoCellDef="let element">
+            <div class="fileDownload" [attr.rel]="element.FileLink" tabindex="0" role="button"
+              attr.aria-label="Download File for {{element.FileName}}"><i class="fa fa-download fa-1x"></i></div>
+          </td>
         </ng-container>
         <tr ukho-header-row *ukhoHeaderRowDef="batch.batchFileDetails.columnData"></tr>
         <tr ukho-row *ukhoRowDef="let row; columns: batch.batchFileDetails.columnData"></tr>
       </table>
+
+
+      <ng-container *ngIf="batch.allFilesZipSize!=null">
```

Comment:
Please change this to a simple 'truthy' condition
```suggestion
      <ng-container *ngIf="batch.allFilesZipSize">
```
Doing this covers undefined and zero